components/camera/align/join.go

```diff
@@ -144,6 +172,27 @@ func (jcd *joinColorDepth) NextPointCloud(ctx context.Context) (pointcloud.Point
 	return jcd.projector.RGBDToPointCloud(rimage.ConvertImage(col), dm)
 }
 
+func (jcd *joinColorDepth) nextPointCloudFromImages(ctx context.Context) (pointcloud.PointCloud, error) {
```

Comment:
If the config has the same depth_camera and color_camera, then assume you can use an `Images` call to get the color and depth image for the merging